src/ai/backend/manager/cli/image_impl.py

```diff
@@ -192,3 +194,58 @@ async def dealias(cli_ctx, alias):
             log.exception("Alias not found.")
             return
         await session.delete(alias_row)
+
+
+async def validate_image_alias(cli_ctx, alias: str) -> None:
+    async with (
+        connect_database(cli_ctx.local_config) as db,
+        db.begin_session() as session,
+    ):
+        try:
+            image_row = await ImageRow.from_alias(session, alias)
+            if image_row is None:
+                raise UnknownImageReference
```

Comment:
```suggestion
            image_row = await ImageRow.from_alias(session, alias)
```
Don't need null-check here because `UnknownImageReference` is raised inside of `ImageRow.from_alias()` function